.scalafmt.conf

```diff
@@ -10,7 +10,6 @@ unindentTopLevelOperators  = true
 align.tokens               = [{code = "=>", owner = "Case"}]
 align.openParenDefnSite    = false
 align.openParenCallSite    = false
-optIn.breakChainOnFirstMethodDot = false
```

Comment:
I find this annoying because it creates chains of method calls and I have to parse characters and params with my eyes to make sense of the code.